src/paginate.ts

```diff
@@ -8,6 +8,9 @@ import { Pagination } from './pagination';
 import { IPaginationOptions } from './interfaces';
 import { createPaginationObject } from './create-pagination';
 
+const DEFAULT_LIMIT = 10;
+const DEFAULT_PAGE = 1;
```

Comment:
I like this 👍  I might add these values to the options input in another PR/refactor :)